cellxgene_schema_cli/cellxgene_schema/atac_seq.py

```diff
@@ -475,8 +479,14 @@ def prepare_fragment(
     :param write_algorithm:
     :return:
     """
+    step_size = 1_000_000_000  # 1 million
```

Comment:
whoa, i didn't know you could just add underscores to python integers and it would work? 😮

Reply:
you can also do scientific notation, but I found the `_` easier to read for numbers less than a billion